src/pages/lib/auth/manageusers/q/platform/[platform].mdx

```diff
@@ -1,11 +1,456 @@
+
+</Accordion>
+
+### Pass user attributes during sign-up
+
+You can create user attributes during or after sign-up. To do this as part of sign-up you can pass them in the `attributes` object of `Auth.signUp` function parameters:
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify'
+
+async function handleSignUp() {
+  try {
+    await Auth.signUp({
+      username: 'jdoe',
+      password: 'mysecurerandompassword#123',
+      attributes: {
+        email: 'me@domain.com',
+        phone_number: '+12128601234', // E.164 number convention
+        given_name: 'Jane',
+        family_name: 'Doe',
+        nickname: 'Jane',
+        'custom:attribute_name': 'attribute_value'
+      }
+    });
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify'
+
+async function handleSignUp() {
+  try {
+    await Auth.signUp({
+      username: 'jdoe',
+      password: 'mysecurerandompassword#123',
+      attributes: {
+        email: 'me@domain.com',
+        phone_number: '+12128601234', // E.164 number convention
+        given_name: 'Jane',
+        family_name: 'Doe',
+        nickname: 'Jane',
+        'custom:attribute_name': 'attribute_value'
+      }
+    });
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
+</Block>
+
+</BlockSwitcher>
+
+### Retrieve user attributes
+
+You can then retrieve user attributes for your users to read in their profile. This helps you personalize their frontend experience as well as control what they will see. Visible attributes will be limited to those you enabled during the `Specify read attributes` configured with the `auth` category. Then you can use the `currentAuthenticatedUser` method of the `Auth` class to read user attributes:
+
+```ts
+const user = await Auth.currentAuthenticatedUser();
+
+const { attributes } = user;
+```
+
+Note that if you need to retrieve the latest user attributes after they are changed, you will need to bypass the local cache like so:
+
+```ts
+const user = await Auth.currentAuthenticatedUser({
+  bypassCache: true
+});
+```
+The `bypassCache` parameter defaults to false. Alternatively, your users can sign out and sign back in to retrieve and view their latest attributes.
+
+## Enable users to update, verify, and delete specific attributes
+
+You can enable your users to update, verify, and delete specific user attributes as their information changes over time. These attributes are enabled as writeable when you update the `Specify write attributes` in your `Auth` configuration.
+
+### Update user attributes
+
+You can enable users to make updates after sign-up by using the `updateUserAttributes` method of the `Auth` class:
```

Comment:
You can enable users to make updates after sign-up by using the `updateUserAttributes` method: